lib/modules/manager/dockerfile/extract.ts

```diff
@@ -283,6 +283,25 @@ export function extractPackageFile(
       instruction += '\n' + lineLookahead;
     }
 
+    const syntaxRegex = regEx('^#[ \\t]*syntax=(?<image>\\S+)', 'm');
```

Comment:
this needs to be moved before line continuation regex above and should be conditional like the escape char finding. because its only allowed once at the very beginning of the file